Update configure-data-collection.md (#1598)
SumoLogic/sumologic-documentation: docs/apm/real-user-monitoring/configure-data-collection.md

import TabItem from '@theme/TabItem';
import Iframe from 'react-iframe';

> To collect [traces](/docs/apm/traces) and RUM mertrics from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you'll put in a script that sends trace data in [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol. Alternatively you can also use an intermediary OTel collector, if you require data to flow over your infrastructure rather than directly to Sumo Logic. Note however this will disable automatic geo-location recognition capabilities.

Reviewer: I'd rephrase it as:
The source will have an endpoint URL that you can use in a script to send [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol trace data.

> To collect [traces](/docs/apm/traces) and RUM mertrics from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you'll put in a script that sends trace data in [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol. Alternatively you can also use an intermediary OTel collector, if you require data to flow over your infrastructure rather than directly to Sumo Logic. Note however this will disable automatic geo-location recognition capabilities.

Reviewer: ```suggestion
To collect [traces](/docs/apm/traces) and RUM metrics from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you can use in a script to send [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol trace data. Alternatively, you can also use an intermediary OTel collector if you require data to flow over your infrastructure rather than directly to Sumo Logic. Note however this will disable automatic geo-location recognition capabilities.
```

> To collect [traces](/docs/apm/traces) and RUM mertrics from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you'll put in a script that sends trace data in [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol. Alternatively you can also use an intermediary OTel collector, if you require data to flow over your infrastructure rather than directly to Sumo Logic. Note however this will disable automatic geo-location recognition capabilities.

Reviewer 2: ```suggestion
To collect [traces](/docs/apm/traces) and RUM metrics from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you'll put in a script that sends trace data in [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol. Alternatively, you can also use an intermediary OTel collector, if you require data to flow over your infrastructure rather than directly to Sumo Logic. Note however this will disable automatic geo-location recognition capabilities.
```

:::

## Prerequisites

> XHR and navigation/route changes support as well as errors collection require RUM script in version 4 or higher (`https://rum.sumologic.com/sumologic-rum-v4.js`). Make sure you're using the correct version in your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control later in this document.

Reviewer: ```suggestion
Support for XHR and navigation/route changes, as well as error collection, require RUM script version 4 or higher ('https://rum.sumologic.com/sumologic-rum-v4.js'). Make sure you're using the correct version in your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control later in this document.
```
```suggestion
XHR and navigation/route changes support, and errors collection require the RUM script in version 4 or higher (`https://rum.sumologic.com/sumologic-rum-v4.js`). Make sure you're using the correct version on your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control later in this document.
```

> XHR and navigation/route changes support, and errors collection require the RUM script in version 4 or higher. To know more, refer to the link `https://rum.sumologic.com/sumologic-rum-v4.js`. Make sure you're using the correct version on your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control later in this document.

Reviewer 2: ```suggestion
To utilize XHR and navigation/route changes, and errors collection, you must use RUM script version 4 or higher (`https://rum.sumologic.com/sumologic-rum-v4.js`). Make sure you're using the correct version in your pages. For automatic updates, use the script `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control [later in this document](#step-2-add-rum-script-to-your-page-header).
```
